typescript/cli/src/deploy/warp.ts

```diff
@@ -885,12 +885,20 @@ async function submitWarpApplyTransactions(
   params: WarpApplyParams,
   chainTransactions: Record<string, AnnotatedEV5Transaction[]>,
```

Comment:
the key of this is a chainId as a string?

Reply:
that's right - but it's because the lodash `groupBy` helper returns a Dictionary whose keys are always strings

Reply:
I did not realize we added lodash
@jmrossy would have a conniption

Reply:
aha yeah idk when it got added but it's been there for a little bit

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/commit/1474865ae14d014a86893d79ac0a36eea43e4a70 I guess he was onboard :)

Reply:
I forgot I was the one who added it 🤣

Reply:
Just be sure to use `lodash-es`!